### ISSUE TITLE
Syncing Multiplayer World don't load

### ISSUE THREAD

**Issue by Fraah** [SubnauticaNitrox/Nitrox#1266]
### Version
**1.3.0.0**

### Bug
Me and my friends have an issue with loading. It stops on 60% and 99%.
It seems like the character is in game but the loading page is on screen.

**Comment by Fraah** [comment]
Syncing Multiplayer World 60%
Running BasePieceMetadataInitialSyncprocessor 99%

It can't load

**Comment by dartasen** [comment]
Hi @Fraah,

Looks like the world save might be corrupted, would you mind sending us a .zip of the world ?

**Comment by Fraah** [comment]
[world.zip](https://github.com/SubnauticaNitrox/Nitrox/files/5423985/world.zip)

**Comment by Logeer** [comment]
I have the same problem we had a lots of stuff can you help me too?

**Comment by dartasen** [comment]
We're working on improving save files, the only fix available for now is to reset the corrupted world

**Comment by ThIsJaCk23657689** [comment]
Hey guys, I encountered this problem while playing with my friends just few hours ago, but I found a workaround. I went to the "PlayerData.json" file in the "world" folder and deleted the "SpawnPosition" and "SpawnRotation" sections for all players. Then restarted the server.  After that I was able to enter the server normally.

**Make sure to back up your world files before attempting to modify them.**
